api/docs/v2/storefront/index.yaml

```diff
@@ -550,7 +550,62 @@ paths:
             application/vnd.api+json:
               schema:
                 $ref: '#/components/schemas/Cart'
-              examples: {}
+              examples:
+                example:
+                  value:
+                    data:
+                      id: '511'
+                      type: cart
+                      attributes:
+                        number: R904373524
+                        item_total: '0.0'
```

Comment:
@pmilewski let's have some positive numbers here also let's include the includes with one for each type - line item, shipment, promotion, payment, addresses, etc